apps/perf-test/tasks/flamegraph/helpers.js

```diff
@@ -0,0 +1,17 @@
+// TODO: test regex on mac, linux and windows
+const frameworkName = new RegExp(
+  /~([a-zA-Z]{1}|[a-zA-Z]{2}|[a-zA-Z]{1}|[a-zA-Z]{1}\.[a-zA-Z]{1})[\. ]|~render |~\(anonymous\)|\(unknown\)|\~webpack/
```

Comment:
What's the first part of this meant to match? Also, I'm pretty sure it can be shorter like this:
```suggestion
  /~([a-zA-Z]{1,2}|[a-zA-Z]{1}\.[a-zA-Z])[\. ]|~render |~\(anonymous\)|\(unknown\)|\~webpack/
```

Reply:
Thanks. I was iterating it but didn't consolidate it.